### PR TITLE
CHE-6910: Use JDT-LS to update maven projects

### PR DESCRIPTION
**Pull request by mmorhun**
### What does this PR do?
Use JDT-LS as backend for updating maven projects.
Moves maven related actions to java LS plugin.

Current implementation uses synchronous requests, but when progress issue with LS will be resolved, it should listen to asynchronous messages about projects updating and update progress bar, percentage etc.

### What issues does this PR fix or reference?
https://github.com/eclipse/che/issues/6910

### CODE REVIEW

**Review by azatsarynnyy — APPROVED** [review]

File: plugins/plugin-java/che-plugin-java-ext-lang-client/src/main/resources/org/eclipse/che/ide/ext/java/client/svg/maven-command-type.svg [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+        </g>
+    </g>
+</svg>
```
Comment (tolusha) [inline plugins/plugin-java/che-plugin-java-ext-lang-client/src/main/resources/org/eclipse/che/ide/ext/java/client/svg/maven-command-type.svg]: new line

**Review by tolusha — APPROVED** [review]

**Review by svor — APPROVED** [review]

File: plugins/plugin-java/che-plugin-java-server/src/main/java/org/eclipse/che/plugin/java/languageserver/JavaLanguageServerExtensionService.java [hunk 2]
```diff
@@ -368,6 +382,46 @@ public String getEffectivePom(String projectPath) {
...
+    CompletableFuture<Object> requestResult =
+        executeCommand(REIMPORT_MAVEN_PROJECTS_COMMAND, singletonList(parameters));
+
+    final List<String> result;
+    Type targetClassType = new TypeToken<ArrayList<String>>() {}.getType();
+    try {
+      result =
+          gson.fromJson(
+              gson.toJson(
+                  requestResult.get(REIMPORT_MAVEN_PROJECTS_REQUEST_TIMEOUT, TimeUnit.SECONDS)),
```
Comment (tsmaeder) [inline plugins/plugin-java/che-plugin-java-server/src/main/java/org/eclipse/che/plugin/java/languageserver/JavaLanguageServerExtensionService.java]: Again, this will wait roughly 16 hours.

File: plugins/plugin-orion/che-plugin-orion-editor/pom.xml [hunk 3]
```diff
@@ -1,44 +0,0 @@
-<?xml version="1.0" encoding="UTF-8"?>
```
Comment (tsmaeder) [inline plugins/plugin-orion/che-plugin-orion-editor/pom.xml]: This seems unrelated....what's happening here?

**Review by tsmaeder — CHANGES_REQUESTED** [review]

File: plugins/plugin-java/che-plugin-java-server/src/main/java/org/eclipse/che/plugin/java/languageserver/JavaLanguageServerExtensionService.java (hunk 2 above)
Comment (mmorhun) [reply plugins/plugin-java/che-plugin-java-server/src/main/java/org/eclipse/che/plugin/java/languageserver/JavaLanguageServerExtensionService.java]: Ha, thanks) my bad

File: plugins/plugin-orion/che-plugin-orion-editor/pom.xml (hunk 3 above)
Comment (mmorhun) [reply plugins/plugin-orion/che-plugin-orion-editor/pom.xml]: Yes, probably this is here occasionally

**Review by tsmaeder — APPROVED** [review]
